src/components/about.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import styled,{css} from 'styled-components';
```

Comment:
`styled` has been imported but not used while `css` has been used to place inline styles. A better implementation, in my opinion, will be to not use `css` and instead build the required styled components using `styled` to improve code readability and also reduce the number of imports. Also, keeping these styles only in the external CSS file is also a good option, given that it has been imported too in line 7.

Reply:
Ok sir